gridstatus/spp.py

```diff
@@ -792,9 +792,17 @@ def get_lmp_real_time_weis(self, date, end=None, verbose=False):
             # folder path is based on start date
             # file name is based on end date
             url = f"{FILE_BROWSER_DOWNLOAD_URL}/lmp-by-settlement-location-weis?path=/{date.strftime('%Y')}/{date.strftime('%m')}/By_Interval/{date.strftime('%d')}/WEIS-RTBM-LMP-SL-{end.strftime('%Y%m%d%H%M')}.csv"  # noqa
+
+        # TODO: sometimes there are missing interval files (example: https://portal.spp.org/pages/lmp-by-settlement-location-weis#%2F2024%2F01%2FBy_Interval%2F21) # noqa
```

Comment:
The data for at least two 5-min intervals is missing on this day. I don't think we have any choice but to catch the error and log a message.

#### Intervals at end of day not present
https://portal.spp.org/pages/lmp-by-settlement-location-weis#%2F2024%2F01%2FBy_Interval%2F21

<img width="887" alt="Screenshot 2024-01-23 at 14 09 40" src="https://github.com/kmax12/gridstatus/assets/17484616/d65ba1d8-5583-43b9-baa4-20db6ca1a0a9">

#### Full Day
https://portal.spp.org/pages/lmp-by-settlement-location-weis#%2F2024%2F01%2FBy_Interval%2F22

<img width="919" alt="Screenshot 2024-01-23 at 14 09 29" src="https://github.com/kmax12/gridstatus/assets/17484616/b8390f7d-1313-4c55-af9f-820cfb8004f6">